app/components/object-inspector/property.js

```diff
@@ -46,6 +44,42 @@ export default Component.extend({
 
   showDependentKeys: and('isDepsExpanded', 'hasDependentKeys'),
 
+  canCalculate: computed('model', 'isCalculated', 'isComputedProperty', function () {
+    if (this.get('isOverridden')) return false;
+    if (!this.get('isComputedProperty') && this.get('model.isGetter') && this.get('model.isExpensive')) {
+      return true;
+    }
+    return this.get('isComputedProperty') && !this.get('isCalculated');
+  }),
+
+  icon: computed('isService', 'model.inspect.value', 'model.isTracked', 'model.isProperty', 'model.isGetter', 'isFunction', function () {
```

Comment:
`icon` seems to be used for displaying strings. Perhaps it would be better named something like `itemType` since it is not always for an icon?